### PR TITLE
Update SSDT-Xhci.asl

### PR DESCRIPTION
**Pull request by Malumen**
Forked from Osy86, changed HS07 and HS08 to be copies of HS04 in the hopes of enabling additional ports from motherboard header cable.

I have no clue what I'm doing. There are a lot of differences with Osy's original code and I don't know how that happened.

### CODE REVIEW

**Comment by Malumen** [comment]
@osy86 I have no idea why my copy+paste from my .asl file is so wildly different from your current code.

File: ACPI/SSDT-Xhci.asl [hunk 1]
```diff
@@ -139,86 +159,85 @@ DefinitionBlock ("", "SSDT", 2, "OSY86 ", "Xhci", 0x00001000)
                     Zero, 
                     Zero
                 })
```
Comment (Malumen) [inline ACPI/SSDT-Xhci.asl:161]: why }) ? where does ) begin?

File: ACPI/SSDT-Xhci.asl [hunk 2]
```diff
@@ -60,77 +76,81 @@ DefinitionBlock ("", "SSDT", 2, "OSY86 ", "Xhci", 0x00001000)
...
             {
-                Method (_STA, 0, NotSerialized)  // _STA: Status
+                Method (_DSM, 4, NotSerialized)  // _DSM: Device-Specific Method
                 {
-                    Return (Zero)
+                    Local0 = Package (0x00){}
+                    DTGP (Arg0, Arg1, Arg2, Arg3, RefOf (Local0))
+                    Return (Local0)
                 }
             }
```
Comment (Malumen) [inline ACPI/SSDT-Xhci.asl:151]: this does not work; this causes HS07 and HS08 to be seen as USB3.0 ports and not recognize anything being plugged in...

**Review by Malumen — COMMENTED** [review]
the changes do not achieve the end-result of enabling HS07 and HS08 to be USB2.0 Type-A ports. In addition, Osy86's original code varies wildly in comparison with current coe found in end-user devices.

**Comment by osy86** [comment]
Fixed with dba685663b2544635603aaf0cca59c05f6040676

**Pull request closed without merging**